assets/js/order-script.js

```diff
@@ -80,6 +80,21 @@ jQuery(document).ready(function($) {
 		});
 	});
 
+	// disable inputs from start
+	$( '.wcpdf-data-fields' ).find( ':input' ).attr( 'disabled', true );
```

Comment:
any idea what's different here compared to what you are already setting server side? because you already set it to disabled there too 🤔 (which should be enough already).
Because I don't see anything different anymore if I'm comparing. If I disabled this whole code block it works fine so I wonder if I was testing incorrectly... Or is it because you added `:visible` to the selector used for serialization?